### ISSUE TITLE
Puro fails to update IntelliJ config

### ISSUE THREAD

**Issue by dsyrstad** [PixelToast/puro#9]
Using puro 1.2.5 and IntelliJ IDEA 2023.1.1 (Ultimate Edition), Build #IU-231.8770.65, built on April 27, 2023

```bash
$ puro use stable
[E] Exception while installing IntelliJ config
[E] Bad state: No element
    #0      Iterable.first (dart:core/iterable.dart:624)
    #1      IntelliJConfig.load (package:puro/src/workspace/intellij.dart:284)
    #2      installIdeConfigs.<anonymous closure> (package:puro/src/workspace/install.dart:28)
    #3      runOptional (package:puro/src/logger.dart:173)
    #4      installIdeConfigs (package:puro/src/workspace/install.dart:24)
    #5      installWorkspaceEnvironment.<anonymous closure> (package:puro/src/workspace/install.dart:107)
    #6      runOptional (package:puro/src/logger.dart:173)
    #7      installWorkspaceEnvironment (package:puro/src/workspace/install.dart:104)
    <asynchronous suspension>
    #8      switchEnvironment (package:puro/src/workspace/install.dart:133)
    <asynchronous suspension>
    #9      EnvUseCommand.run (package:puro/src/commands/env_use.dart:83)
    <asynchronous suspension>
    #10     CommandRunner.runCommand (package:args/command_runner.dart:212)
    <asynchronous suspension>
    #11     main (package:puro/src/cli.dart:234)
    <asynchronous suspension>
    
[✓] Switched to environment `stable`
```

This leaves .idea/libraries/Dart_SDK.xml as:
```
<component name="libraryTable">
  <library name="Dart SDK">
    <CLASSES/>
    <JAVADOC/>
    <SOURCES/>
  </library>
</component>
```

And  creates .idea/libraries/Dart_Packages.xml.bak, but doesn't create a new Dart_Packages.xml.

**Comment by PixelToast** [comment]
Thanks for the bug report!

I'm at furcon but I'll take a look as soon as I can, the way the flutter/dart plugin is configured in IntelliJ is a big mess :(

**Comment by PixelToast** [comment]
This was indeed a bug in Puro, it should be fixed in 1.2.6, thanks again!
